features/gba-dev/kinderen.feature

```diff
@@ -0,0 +1,146 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Kinderen van een persoon raadplegen
+  Van een persoon worden -indien gevraagd met de fields parameter- de kinderen geleverd.
+  Dit bevat enkele identificerende eigenschappen van de kinderen.
+
+  Rule: de actuele gegevens van kinderen worden geleverd
+
+    Scenario: de naam van een kind is gecorrigeerd
```

Comment:
dit issue was zo complex, omdat een GBA-V proefomgeving persoon gebruikt werd. Nu we gekozen hebben zelf data toe te voegen, kunnen we dit beperken tot alleen benodigde gegevens. Hieronder zouden dat alleen de eerste twee Gegeven stappen zijn.

En de enige Dan stap wordt dan:
```
Dan heeft de response een persoon met een 'kind' met de volgende gegevens
       | naam               | waarde  |
       | naam.voornamen     | William |
       | naam.voorvoegsel   | de      |
       | naam.geslachtsnaam | Vries   |
```

Reply:
Is aangepast.

Reply:
Ik neem aan dat dit ook geldt voor hetzelfde scenario in 'features/kinderen.feature'.